packages/core/src/repositories/evaluationsRepository.ts

```diff
@@ -1,25 +1,48 @@
-import { eq, getTableColumns } from 'drizzle-orm'
+import { omit } from 'lodash-es'
 
+import { and, eq, getTableColumns, sql } from 'drizzle-orm'
+
+import { EvaluationMetadataType } from '../constants'
 import { NotFoundError, Result } from '../lib'
-import { evaluations } from '../schema'
+import { evaluations, llmAsJudgeEvaluationMetadatas } from '../schema'
 import Repository from './repository'
 
-const tt = getTableColumns(evaluations)
+const tt = {
+  ...getTableColumns(evaluations),
+  metadata: {
+    id: sql<number>`llm_as_judge_evaluation_metadatas.id`.as(
+      'metadata_metadata_id',
+    ),
+    ...omit(getTableColumns(llmAsJudgeEvaluationMetadatas), [
+      'id',
+      'metadataType',
+      'createdAt',
+      'updatedAt',
+    ]),
+  },
+}
```

Comment:
since we will always want the metadata with the evaluation i've opted for including it in the scope return value so you can always do `evaluation.metadata` bonus point:  it will force u to use the repository which will enforce tenancy as a nice side effect